addons/website_sale/static/src/js/website_sale.js

```diff
@@ -97,8 +97,15 @@ export const WebsiteSale = publicWidget.Widget.extend(VariantMixin, cartHandlerM
         });
 
         // This allows conditional styling for the filmstrip
+        const filmstripContainer = this.el.querySelector('.o_wsale_filmstip_container');
+        const filmstripWrapper = this.el.querySelector('.o_wsale_filmstip_wrapper');
+
         if (isBrowserFirefox() || hasTouch()) {
-            this.el.querySelector('.o_wsale_filmstip_container')?.classList.add('o_wsale_filmstip_fancy_disabled');
+            filmstripContainer?.classList.add('o_wsale_filmstip_fancy_disabled');
+        }
+        // Disable the margin on hover if there is no scrollbar
+        if (filmstripWrapper?.getBoundingClientRect().width < filmstripContainer?.getBoundingClientRect().width) {
```

Comment:
You can add this condition in above if :thinking:

Reply:
Yes but it was making it unreadable. This looks like a more clean code.
By keeping it different, we can understand the conditions better I think. :thinking:

Reply:
Instead you can divide your condition per line or something like that
Because in future if I have to do some other operation on those condition then I have to write them twice or may be add new condition then I might not be sure in which block should I put it
You can add in line comment
But we should not add multiple if condition for same operation